### PR TITLE
Fetch sentiment analysis info and display it

### PR DESCRIPTION
**Pull request by jeinygroove**

### CODE REVIEW

**Review by sashapff — APPROVED** [review]

**Review by RazvanBudacaTechquarter — APPROVED** [review]
Maybe we should make the info icons a bit smaller.

**Comment by jeinygroove** [comment]
> Maybe we should make the info icons a bit smaller.

Yes, thought about it, will make it a little bit late.

File: src/main/webapp/src/components/sentimentAnalysisInfo.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        <div>
+          <p>{this.state.error.message}</p>
+        </div>
+      );
+    }
+
+    if (this.state.isLoading) {
+      return (
+        <div>
+          <CircularProgress style={{ color: "black" }} />
```
Comment (amaurymedeiros) [inline src/main/webapp/src/components/sentimentAnalysisInfo.js:87]: nit: can we centralize the style of the CircularProgress somewhere? We should be consistent across all the parts of the app, so there is no need to set `"color: black"` everywhere, because if we change our mind we need to replace it everywhere.

This change should not be part of this Pull Request, since it's unrelated.

**Review by amaurymedeiros — COMMENTED** [review]
LGTM

File: src/main/webapp/src/components/sentimentAnalysisInfo.js (hunk 1 above)
Comment (jeinygroove) [reply src/main/webapp/src/components/sentimentAnalysisInfo.js:87]: sure

**Review by jgaillard85 — APPROVED** [review]
